src/admin/containers/email.js

```diff
@@ -35,10 +36,18 @@ const Email = (
     >
       { map(send_button, zip(email_ids, [sub, news, remind, custom])) }
     </form>
-    {custom_emails ? <CustomEmailForm submit={submit_custom_email} members={custom_emails.members}/>
-  : keys(emails).length > 0 && email_list({ emails, submit_email, ...list_props })}
+    {email_sent
+      ? <EmailNotification email_sent={email_sent}/>
```

Comment:
At the moment it only gets more nested 😟  in later work

But we can change that (today?) in the bounced emails branch so will leave this in for now.